proposals/lip-0065.md

```diff
@@ -206,12 +206,12 @@ For events emitted during the command execution, modules specify whether the eve
 
 ### Events Tree
 
-Events are inserted into the events tree once for each entry in the event `topics` array. The leaf key is the concatenation of the hash of the element in the topics array, the event `index` converted to a binary string of length `EVENT_INDEX_LENGTH_BITS` and concatenated with the topic index of `TOPIC_INDEX_LENGTH_BITS` bits (corresponding to the entry in the `topics` array), for a total of `TOTAL_INDEX_LENGTH_BYTES` bytes. The leaf value is the serialization of the event properties using the `eventSchema` given above.
+Events are inserted into the events tree once for each entry in the event `topics` array. The leaf key is the concatenation of the hash of the element in the topics array, the event `index` converted to a binary string of length `EVENT_INDEX_LENGTH_BITS` and concatenated with the topic index of `TOPIC_INDEX_LENGTH_BITS` bits (corresponding to the entry in the `topics` array), for a total of `TOTAL_INDEX_LENGTH_BYTES` bytes. The leaf value is the SHA-256 hash of the serialization of the event properties using the `eventSchema` given above.
 
 In summary, for each event one key-value pair is added to the events tree for each entry `topic` in the associated `topics` array, where:
 
-* `key = SHA-256(topic)[:TOPIC_HASH_LENGTH_BYTES] + ((event index << TOPIC_INDEX_LENGTH_BITS) + topic index).to_bytes(TOTAL_INDEX_LENGTH_BYTES, byteorder="big")`
-* `value` = serialization of event according to `eventSchema`.
+* `key = sha256(topic)[:TOPIC_HASH_LENGTH_BYTES] + ((event index << TOPIC_INDEX_LENGTH_BITS) + topic index).to_bytes(TOTAL_INDEX_LENGTH_BYTES, byteorder="big")`
```

Comment:
In the constants table and in `eventID()` the hash it is denoted by `SHA-256()` and here `sha256()`. I suggest using the same notation accross the whole LIP for consistency.

Reply:
fixed it in the code, I left it as `SHA-256` in the text (we do the same in other LIPs)